salt/modules/mac_brew_pkg.py

```diff
@@ -258,7 +258,10 @@ def latest_version(*names, **kwargs):
 
     def get_version(pkg_info):
         # Perhaps this will need an option to pick devel by default
-        return pkg_info["versions"]["stable"] or pkg_info["versions"]["devel"]
+        version = pkg_info["versions"]["stable"] or pkg_info["versions"]["devel"]
+        if pkg_info["versions"]["bottle"] and pkg_info["revision"] >= 1:
```

Comment:
will "bottle" always be included in `pkg_info["versions"]` or should we be calling a `.get()` here?

Reply:
just ran the numbers and it looks like bottle is included in every formula's versions. in fact it looks like currently it is only false in 1 formula. 

```
> brew info --all --json=v2 | jq ".formulae[].name" | wc -l
6250
> brew info --all --json=v2 | jq ".formulae[].versions.bottle" | sort | uniq -c
      1 false
   6249 true
```